packages/devtools_app/test/test_infra/scenes/standalone_ui/editor_sidebar.dart

```diff
@@ -77,10 +76,12 @@ class EditorSidebarScene extends Scene {
 
     // We assume a DTD is available on 8500. There's a VS Code task that
     // launches this as part of the standalone_ui/editor_sidebar config.
+    // TODO(dantup): Add a way for the mock editor to set workspace roots so
+    //  the extensions parts can work in the sidebar.
```

Comment:
Multiline TODOs are indented by a space because that's how the analyzer knows the next line is part of the TODO and not an unrelated comment. If we remove this, the TODO diagnostics will be truncated. It might not matter too much here if it's unlikely anyone is using the TODO diagnostics though, so I'll do whichever you prefer!